sdk/storage/Azure.Storage.Blobs/samples/README.md

```diff
@@ -1,20 +1,20 @@
 ---
 page_type: sample
 languages:
-- .NET
+- dotnet
 products:
 - azure
 - azure-storage
 ---
 
-# Azure Storage Blob client library samples for .NET
+# Azure Storage Blob client library samples for dotnet
 
-These sample programs show how to use the .NET client libraries for Azure Storage Blobs in some common scenarios.
+These sample programs show how to use the dotnet client libraries for Azure Storage Blobs in some common scenarios.
```

Comment:
.NET should be used here, not dotnet.  The slug in the front-matter needs to be dotnet